typings/query-helpers.d.ts

```diff
@@ -20,7 +20,7 @@ export type AllByAttribute = (
 
 export const queryByAttribute: QueryByAttribute
 export const queryAllByAttribute: AllByAttribute
-export const debugDOM: (htmlElement: HTMLElement) => string
+export const logDOM: (htmlElement: HTMLElement) => void
```

Comment:
This is the breaking change.

Impact is minimal because people probably aren't relying on this for their actual tests and it's more of a workflow method (which I doubt many people use).

That said, some libraries built on top of DTL may use this method, and the upgrade strategy for them is to switch from `debugDOM` to `prettyDOM`.